### PR TITLE
pdiem fr Translate 

### PR DESCRIPTION
**Pull request by tionebDotsamapha**
…iem.md

### CODE REVIEW

File: content/docs/_index.fr-fr.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ Phala Network s'attaque à ce problème difficile. Il cherche à fournir de soli
 
 
 {{< button "poc4" "Miner sur Phala" >}}
-{{< button "spec" "Spécifications Techniques" >}}
+{{< button "pdiem" "Apprendre pDiem" >}}
+{{< button "spec" "Sécifications Techniques" >}}
```
Comment (krhougs) [inline content/docs/_index.fr-fr.md]: last typo lol

File: content/docs/pdiem/contract-implementation.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: "Mise en œuvre du contrat"
+weight: 5
+draft: false
+---
+
+Le contrat pdiem implémente un client léger et un portefeuille Diem. D'une part, les relayeurs pDiem communiquent avec le contrat pour synchroniser les transactions entre le côté Diem et le côté pDiem. D'autre part, il fournit l'interface permettant aux utilisateurs de gérer leurs actifs cross-chain.
```
Comment (krhougs) [inline content/docs/pdiem/contract-implementation.fr-fr.md]: use `pDiem` instead of `pdiem`

**Comment by krhougs** [comment]
Merci beaucoup !

**Comment by tionebDotsamapha** [comment]
👍 
Thank you @krhougs for your review.

**Comment by tionebDotsamapha** [comment]
hi everyone.
That's possible to merge the pull request?
Another is ready : develloper. 

Have a good day !

**Comment by h4x3rotab** [comment]
Thank you very much. Looks good to me

**Comment by h4x3rotab** [comment]
Published! Waiting for Github Pages to load it

**Pull request merged**